pkgs/by-name/zu/zutty/package.nix

```diff
@@ -0,0 +1,47 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, pkg-config
+, freetype
+, wafHook
+, python3
+, libXmu
+, glew-egl
+, ucs-fonts
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "zutty";
+  version = "0.14";
+
+  src = fetchFromGitHub {
+    owner = "tomscii";
+    repo = "zutty";
+    rev = finalAttrs.version;
+    hash = "sha256-b/q7hIi/U/GkKo+MIFX2wWnHZAy5rQGXNul3I1pxo1Q=";
+  };
+
+  postPatch = ''
+    substituteInPlace src/options.h \
+      --replace /usr/share/fonts ${ucs-fonts}/share/fonts
+  '';
```

Comment:
I'd probably add a `isNixOS ? true` and set the path to `/run/current-system/sw/share/X11/fonts` accordingly.

Reply:
If we set the font path to /run/current-system/sw/share/X11/fonts, isn't there a risk that the ucs-fonts won't be installed, and zutty will fail to start as it can't find it's default font (9x18)?

I'm really not sure the right way to package zutty here. Perhaps make it so you can configure both the default font and the font path?